src/Generators/Microsoft.Gen.Logging/Emission/Emitter.Method.cs

```diff
@@ -134,6 +134,22 @@ private void GenLogMethod(LoggingMethod lm)
         OutLn($"{stateName}.Clear();");
         OutCloseBrace();
 
+        /// <summary>
+        /// Returns a non-randomized hash code for the given string.
+        /// We always return a positive value.
+        /// </summary>
+        static int GetNonRandomizedHashCode(string s)
```

Comment:
Minor: consider adding a comment mentioning that the code is replicating the behavior of https://github.com/dotnet/runtime/blob/993b23fc101a062be87de9dac21302034abd337d/src/libraries/Microsoft.Extensions.Logging.Abstractions/gen/LoggerMessageGenerator.Parser.cs#L831.

Reply:
Done.